charts/country-risk/values.yaml

```diff
@@ -0,0 +1,367 @@
+  nodeSelector: {}
+
+  # -- Pod toleration constraints
+  tolerations: []
+
+  # Following Catena-X Helm Best Practices @url: https://catenax-ng.github.io/docs/kubernetes-basics/helm
+  # @url: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 100
+          podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+                  # -- Match Pod rules
+                - key: app.kubernetes.io/name
+                  operator: DoesNotExist
+            # -- Key that is used to determine the topology of the cluster
+            topologyKey: kubernetes.io/hostname
+
+  # Following Catena-X Helm Best Practices @url: https://catenax-ng.github.io/docs/kubernetes-basics/helm
+  # @url: https://github.com/helm/charts/blob/master/stable/nginx-ingress/values.yaml#L210
+  # -- Determines if a pod is still alive or not
+  livenessProbe:
+    failureThreshold: 3
+    # -- Number of seconds to wait before performing the first liveness probe
+    initialDelaySeconds: 10
+    # -- Number of seconds to wait between consecutive probes
+    periodSeconds: 1000
+    # -- Number of consecutive successful probes before a pod is considered healthy
+    successThreshold: 1
+    # -- Number of seconds after which a liveness probe times out
+    timeoutSeconds: 1000
+  # -- Determine when a pod is ready to start accepting requests
+  readinessProbe:
+    failureThreshold: 3
+    initialDelaySeconds: 10
+    periodSeconds: 1000
+    successThreshold: 1
+    timeoutSeconds: 1000
+
+  # -- Defines the client secret and client ID
+  applicationSecret:
+    enabled: false
+
+  configmap:
+    # Specifies whether a configmap should be created
+    create: true
+
+
+  elastic:
+    # -- Should elastic be enabled or not
+    enabled: false
+    security:
+      # -- Information about the transport layer security (TLS)
+      tls:
+        # -- Encryption for the REST requests made to the Elastic cluster
+        restEncryption: false
+
+country-risk-backend:
+
+  replicaCount: 1
+
+  image:
+    registry: "ghcr.io"
+    # -- Name of the docker image
+    name: "catenax-ng/tx-vas-country-risk-backend"
+    pullPolicy: Always
+    # -- Overrides the image tag whose default is the chart appVersion.
+    tag: ""
+
+  certificate:
+    # -- Hostname for the certificate
+    host: "backend.countryrisk.example.net"
+
+  # -- Name of the backend service
+  appName: vas-country-risk-backend
+
+  # -- List of secrets to be used
+  imagePullSecrets: []
+  # -- Annotations to be added to the running pod
+  podAnnotations: {}
+
+  # -- Configuration for security-related options of the running pod
+  podSecurityContext:
+    # -- Set the file system group ID for all containers in the pod
+    fsGroup: 2000
+
+  # Spring profiles under which the application should run
+  springProfiles:
+    - dev
```

Comment:
Is this a orphaned value? Did not find it in any Helm template file being used...

If it's orphaned, please remove section also in sub chart `values.yaml` files.

Reply:
done